common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/model/vrang/VraNgPackageDescriptor.java

```diff
@@ -118,6 +119,10 @@ public void setCatalogEntitlement(List<String> catalogEntitlement) {
         this.catalogEntitlement = catalogEntitlement;
     }
 
+	public VraNgPolicy getPolicy() {
+        return this.policy;
+    }
+
```

Comment:
Following the existing structure, can we have both getter and setter?

Reply:
Hi Ivo, I have added a test case for RestClientVraNgprimitive to compare the content of API response. I have already removed few properties which are incompatible. Please let me know, if you need any other properties that needs to be removed. Also, I have added the setter property for setpolicy().